src/python/pants/backend/project_info/peek.py

```diff
@@ -47,6 +53,16 @@ class PeekSubsystem(Outputting, GoalSubsystem):
         help="Whether to leave off values that match the target-defined default values.",
     )
 
+    include_dep_rules = BoolOption(
```

Comment:
Not sure we need this option. Why not always include this data? We don't let you opt in or out of anything else peek shows, it is exhaustive by design. And I don't think there are performance implications?

Reply:
There's an option to opt out of showing fields with defaults values. But I can drop this option if it feels redundant/superfluous. :)

Reply:
I'm always a bit careful when it comes to changing the schema of things.. but since it just adds new fields without changing existing ones, I guess this may be a bit too careful ;)

Reply:
That's about now showing information that may be redundant and very verbose.

But this is "real" data... And I'm not sure we want to open the can of worms of switching data on and off? I'm assuming the performance hit is negligible?

Reply:
yes, perf wise should be fine, as this will have been calculated regardless already.

Reply:
re-introduced option again (https://github.com/pantsbuild/pants/pull/18112#issuecomment-1426405134)